doc/nb/keck_lris_blue_long_400_3400_d560.cfg

```diff
@@ -0,0 +1,37 @@
+[run]
+    ncpus = 1
+    verbosity = 2
+    overwrite = True
+    sortdir = lris_blue_long_400_3400_d560
+
+[rdx]
```

Comment:
The reason I made this 'rdx' instead of 'reduce' is because there is a python method 'reduce', and I wanted to avoid any complications.  I can change this back to reduce to test if any complications actually exist, but I try to avoid giving anything the same name as a python system object.